### PR TITLE
[ACS-1885] libraries version bump

### PR DESCRIPTION
**Pull request by kcichonczyk**
Bumped versions of libraries to prevent security vulnerabilities:
* 'keycloak' version to 15.0.1
* 'mybaitis' version to 3.5.7

### CODE REVIEW

File: repository/src/main/java/org/alfresco/ibatis/SqlSessionMetricsWrapper.java [hunk 1]
```diff
@@ -192,6 +193,24 @@ private void reportQueryExecuted(final long startTime, final String queryTypeTag
         }
     }
 
+    @Override
+    public <T> Cursor<T> selectCursor(String s)
+    {
+        return null;
```
Comment (jamalkm) [inline repository/src/main/java/org/alfresco/ibatis/SqlSessionMetricsWrapper.java:199]: This requires implementation. See other methods in this class how they delegate to the default implementation of mybatis.

File: repository/src/main/java/org/alfresco/ibatis/SqlSessionMetricsWrapper.java [hunk 2]
```diff
@@ -192,6 +193,24 @@ private void reportQueryExecuted(final long startTime, final String queryTypeTag
...
+    }
+
+    @Override
+    public <T> Cursor<T> selectCursor(String s, Object o)
+    {
+        return null;
```
Comment (jamalkm) [inline repository/src/main/java/org/alfresco/ibatis/SqlSessionMetricsWrapper.java:205]: Same as above. Requires implementation.

File: repository/src/main/java/org/alfresco/ibatis/SqlSessionMetricsWrapper.java [hunk 3]
```diff
@@ -192,6 +193,24 @@ private void reportQueryExecuted(final long startTime, final String queryTypeTag
...
+    }
+
+    @Override
+    public <T> Cursor<T> selectCursor(String s, Object o, RowBounds rowBounds)
+    {
+        return null;
```
Comment (jamalkm) [inline repository/src/main/java/org/alfresco/ibatis/SqlSessionMetricsWrapper.java:211]: Same here. Requires implementation.

**Review by jamalkm — CHANGES_REQUESTED** [review]
I left a few comments. However, to summarise:

- Keycloak client shouldn't be upgraded yet
- mybatis overridden methods should delegate to the default implementation rather simply returning `null`

File: repository/src/main/java/org/alfresco/ibatis/SqlSessionMetricsWrapper.java (hunk 3 above)
Comment (kcichonczyk) [reply repository/src/main/java/org/alfresco/ibatis/SqlSessionMetricsWrapper.java]: Implemented.

File: repository/src/main/java/org/alfresco/ibatis/SqlSessionMetricsWrapper.java (hunk 1 above)
Comment (kcichonczyk) [reply repository/src/main/java/org/alfresco/ibatis/SqlSessionMetricsWrapper.java]: Implemented.

File: repository/src/main/java/org/alfresco/ibatis/SqlSessionMetricsWrapper.java (hunk 2 above)
Comment (kcichonczyk) [reply repository/src/main/java/org/alfresco/ibatis/SqlSessionMetricsWrapper.java]: Implemented.

**Comment by alandavis** [comment]
The base issue is now closed

**Pull request closed without merging**